website/docs/cdktf/create-and-deploy/terraform-cloud.mdx

```diff
@@ -74,15 +74,13 @@ First use `cdktf login` to log into Terraform Cloud or your Terraform Enterprise
   </Tab>
 </Tabs>
 
-Run `cdktf init` without passing the `--local` flag.
-This creates a new template CDKTF project for your chosen programming language.
-It will be the basis for a new application with the necessary file structure for you to start defining infrastructure.
-Choose a project language and provide a project name and description.
-Select a Terraform Cloud organization and a name for your workspace and let CDKTF CLI create it for you.
-You can choose whether to send crash reports to the CDKTF team and select the providers you want to use.
+1. Run `cdktf init` without passing the `--local` flag. This creates a new template CDKTF project for your chosen programming language. It will be the basis for a new application with the necessary file structure for you to start defining infrastructure. If you're using Terraform Enterprise, you must also pass the `--tfe-hostname` flag with the hostname of your enterprise instance.
+2. Choose a project language and provide a project name and description.
+3. Select a Terraform Cloud organization and a name for your workspace. CDKTF will create a [CloudBackend](/cli/cloud/settings#the-cloud-block) in your project.
```

Comment:
Tf org can fail if the instance is special, maybe we should add a paragraph Joe just adding the cloud backend in itself would work, at least mid term

Reply:
Can you explain this a bit more?

Reply:
Ah nevermind, forgot that we removed the API call to create the workspace beforehand